develop/webservices/WS_v1/WS_CustomDataTypes_v1/DMAParameterPosition.md

```diff
@@ -4,8 +4,9 @@ uid: DMAParameterPosition
 
 # DMAParameterPosition
 
-| Item   | Format  | Description                                                             |
-|--------|---------|-------------------------------------------------------------------------|
-| Page   | String  | The Data Display page on which the parameter is situated.               |
-| Row    | Integer | The row of the Data Display page on which the parameter is situated.    |
-| Column | Integer | The column of the Data Display page on which the parameter is situated. |
+| Item       | Format  | Description                                                             |
+|------------|---------|-------------------------------------------------------------------------|
+| Page       | String  | The Data Display page on which the parameter is situated.               |
+| PageWeight | Integer |                                                                         |
```

Comment:
@wimme , can you check this one?

Reply:
The order in which the parameter pages should be displayed to the user (as configured in the protocol).